### ISSUE TITLE
VRS: label smoothing

### ISSUE THREAD

**Issue by torbjorgen** [Clinical-Genomics/rdds#299]
Adding label smoothing has a positive impact on model validation performance.

`tf.keras.losses.binary_crossentropy(y_true=y_true, y_pred=y_pred, from_logits=False, label_smoothing=0.125)`

Update model and retrain.
